homeassistant/components/todoist/calendar.py

```diff
@@ -182,6 +191,15 @@ def handle_new_task(call: ServiceCall) -> None:
             label_ids = [label_id_lookup[label.lower()] for label in task_labels]
             item.update(labels=label_ids)
 
+        if ASSIGNEE in call.data:
+            task_assignee = call.data[ASSIGNEE]
```

Comment:
We should lowercase this so that it matches as expected regardless of the case the user entered.  I didn't think this was working until I tried my name in all lowercase, then the task was created successfully with my user assigned.

Reply:
Oh yes I missed to add that again. Will do so.